libkbfs/folder_branch_ops.go

```diff
@@ -3361,7 +3382,14 @@ func (fbo *folderBranchOps) undoMDUpdatesLocked(ctx context.Context,
 		// iterate the ops in reverse and invert each one
 		ops := rmd.data.Changes.Ops
 		for j := len(ops) - 1; j >= 0; j-- {
-			fbo.notifyOneOpLocked(ctx, lState, invertOpForLocalNotifications(ops[j]), rmd)
+			io, err := invertOpForLocalNotifications(ops[j])
+			if err != nil {
```

Comment:
Might be worth a comment here, or something in the error message, explaining that open file handles (i.e., nodes still in the NodeCache) might be incorrect now, and a KBFS restart (or otherwise closing all open file handles) might be needed to get everything working again.